addons/vehicles/XEH_postInit.sqf

```diff
@@ -1,7 +1,38 @@
 // by esteldunedain
 #include "script_component.hpp"
 
+// eject destroyed vehicle init
+private _ejectDestroyedClasses = [];
+{
+    private _ejectDestroyedClass = configName _x;
+    if (-1 == _ejectDestroyedClasses findIf {_ejectDestroyedClass isKindOf _x}) then {
+        TRACE_1("init ejectDestroyed class",_ejectDestroyedClass);
+        _ejectDestroyedClasses pushBack _ejectDestroyedClass;
+        [_ejectDestroyedClass, "init", {
+            params ["_vehicle"];
+            if (!alive _vehicle) exitWith {};
+            TRACE_2("init ejectDestroyed vehicle",_vehicle,typeOf _vehicle);
+            _vehicle addEventHandler ["HandleDamage", {
+                params ["_vehicle", "", "", "", "_ammo"];
+                if (!alive _vehicle) then {
```

Comment:
`if (alive _vehicle) exitWith`?
get's rid of indentation (readability)
and get's rid of the `!` (performance)